### PR TITLE
cassandra-reaper enables web UI

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Review by dlorenc — DISMISSED** [review]

File: cassandra-reaper.yaml [hunk 1]
```diff
@@ -46,7 +46,8 @@ pipeline:
       source /usr/share/nvm/nvm.sh
       nvm install v12
       nvm use v12
-      mvn clean package -DskipTests
+      MAVEN_OPTS="-Xmx384m" mvn -B install -DskipTests -Drat.skip=true
+      # mvn clean package -DskipTests
```
Comment (imjasonh) [inline cassandra-reaper.yaml:50]: Can we just remove this line? Do we expect to need it back soon?

Comment (developer-guy) [reply cassandra-reaper.yaml:50]: let me remove it

**Review by imjasonh — APPROVED** [review]

File: cassandra-reaper.yaml (hunk 1 above)
Comment (mamccorm) [reply cassandra-reaper.yaml]: What does the `-Drat.skip=true` argument do, out of interest? Looking at their CI scripts I see the rest of the arguments, but not that one:
- https://github.com/thelastpickle/cassandra-reaper/blob/da5096f71cd25558bd60e26e830c29e78d0e1160/.github/workflows/ci.yaml#L522

Comment (developer-guy) [reply cassandra-reaper.yaml]: I'm not sure but the build was failing due to rat plugin check, it think its something like compliance checking : https://creadur.apache.org/rat/apache-rat-plugin/

**Pull request merged**